enhancements/builds/remove-jenkins-playload.md

```diff
@@ -0,0 +1,225 @@
+
+### Goals
+
+* Move Jenkins out of the OCP Payload.
+
+### Non-Goals
+
+* Document how JenkinsPipeline strategy builds can be migrated off of OpenShift.
+* Migrate Jenkins users to OpenShift Pipelines/Tekton.
+* Stop distributing the Jenkins image and our associated OpenShift plugins.
+* Disable the JenkinsPipeline build strategy.
+
+## Proposal
+
+### User Stories
+
+As a developer using Jenkins on OpenShift
+I want to continue using my Jenkinsfiles
+So that I can continue to use Jenkins for my CI/CD processes
+
+As an OpenShift release engineer
+I want to remove Jenkins from the OCP payload
+So that I can reduce the size of the payload
+And release Jenkins fixes on their own cadence
+
+### Implementation Details/Notes/Constraints [optional]
+
+#### Remove From Payload
+
+Jenkins and its related agent images will be removed from the OCP payload.
+The delivery of these container images will be replaced with a CPaaS-based release pipeline.
+Jenkins images will be published to the Red Hat Container Catalog, just like any other layered product.
+
+Once the CPaaS pipeline is ready, the Jenkins templates and imagestreams delivered in OpenShift via Samples operator will need to reference the container catalog images (just like other layered products).
+The Samples Operator can then remove its special logic for Jenkins imagestreams and templates.
+
+#### Jenkins Image Tagging Scheme
+
+The Jenkins image will use the maximum supported OCP version as the basis of its tagging scheme:
+
+1. `latest` - latest image build
+2. `4` - latest image that supports OCP 4
+3. `4.<Y>`- latest image that supports up to OCP 4.Y
+4. `4.<Y>.<Z>` - latest image that supports up to OCP 4.Y, with bugfix advisory Z
+
+#### Jenkins Plugin Version Scheme
+
+Jenkins plugins for OpenShift currently use the 1.0.<z> versioning scheme.
+This versioning scheme will be updated so that the minor Y version corresponds to the lowest supported version of OCP 4 for the plugin.
+For example, the next release of the openshift-client plugin should be 1.6.0, since OCP 4.6 is the lowest supported version of OCP for the plugin (that we can support).
+
+The main/master branch for Jenkins and its plugins will be assumed to support the version of OpenShift under development.
+If a breaking change is introduced, a separate branch named `ocp-4.n` should be created to handle bug backports, with `n` representing the highest supported version of OpenShift.
+The requisite delivery pipelines will also need to be updated so that bugfix builds can be issued from `ocp-*` branches.
+
+#### Jenkins / OCP Compatibility
+
+The label `io.openshift.versions` will be added to the Jenkins image via a Dockerfile LABEL instruction.
+Its value will be an inclusive range of OCP major+minor versions that the image is officially supported on.
+For example, the current Jenkins image would set the value `v4.6-v4.8`.
```

Comment:
This is useful metadata to have for users who pull by SHAs. An image tagged `4.6.z` could run on OCP 4.8 just fine, and vice versa.